Update collections_installing.rst

patrik93 opened the pull request:
Adding the `Removing a collection` part to the documentation. #399

patrik93 committed:
Update collections_installing.rst

Adding the `Removing a collection` part to the documentation

oraNod reviewed (commented):
@patrik93 Thanks for the docs contribution! How about adding an example with the command line to make this a bit more complete?

I also see three instances of the word "remove" which might be a bit too repetitive. Consider changing the first clause with something like "If you no longer need a collection, simply remove the installation directory from your filesystem."

patrik93 committed:
Update collections_installing.rst

Rephrase the step description and copying the example command from https://github.com/ansible/ansible/issues/67759#issuecomment-827899471

patrik93 commented:
@oraNod thanks for your feedback, I have applied your suggestion. Feel free to reiterate with your comments on my changes.

oraNod reviewed (approved):
LGTM. An additional minor nit is to use lowercase for "operating system". Thanks @patrik93

patrik93 committed:
Update collections_installing.rst

Make `Operating System` lower case

patrik93 commented:
@oraNod changed to lowercase.

samccann commented:
Thanks @patrik93 for the Ansible docs fix!
